lib/paper_trail/reifier.rb

```diff
@@ -60,10 +61,14 @@ def reify(version, options)
       private
 
       # Set all the attributes in this version on the model.
-      def reify_attributes(model, version, attrs)
+      def reify_attributes(model, version, attrs, options)
         enums = model.class.respond_to?(:defined_enums) ? model.class.defined_enums : {}
         model.class.unserialize_attributes_for_paper_trail! attrs
 
+        # Reifies habtm if the option is passed, otherwise sets the habtm
+        # attributes to nil so they are not reified
+        reify_has_and_belongs_to_many(model, attrs, options)
```

Comment:
All of the other association types (has_one, has_many, and belongs_to) happen in `reify_associations`. Why should HABTM happen here, in `reify_attributes`? So that it can "[set] .. the attributes to nil so they are not reified"? Can you explain that further, please?